source/meetings/2022/april/index.html.md

```diff
@@ -0,0 +1,82 @@
+from _6:20pm_ to _8:00pm_ (meeting starts at _6:30pm_).
+
+[Full registration details are given below](#apr22registration), but do
+note that **we will only be sending out the zoom meeting url to people who
+have registered, because we have limited places, so please do it.**
+
+## Agenda
+
+### Using React in a Ruby Project to Dynamically Generate PDF Documents
+
+[Panos Matsinopoulos](https://getlavanda.com) says:
+
+
+> Fjust needs to be a couple of sentences really, not a
+> full abstract. We use markdown so feel free to format it or embed links for
+> the writeup.
+> 
+> In the Ruby world, we traditionally address the PDF generation problem
+> using gems like [Prawn](https://github.com/prawnpdf/prawn) and [PDFKit](https://github.com/pdfkit/pdfkit)
+> or libraries like [whtmltopdf](https://wkhtmltopdf.org/).
+> 
+> Recently, in one of our Ruby on Rails projects in which we 
+> wanted to generate PDF documents for invoices, we decided to use
+> another programming language and technology: React and AWS Lambda.
+> 
+> In this talk, we will be covering how we did it, what were the challenges
+> and what pros and cons over the incumbent tools for Ruby.
+
+{::coverage year="2022" month="april" talk="using-react-in-a-ruby-project-to-dynamically-generate-p-d-f-documents" /}
+
+### A parse parse pitch: using JSON and custom parsers to create efficient flexible data structures.
+
+[Eleanor McHugh](http://slideshare.net/feyeleanor) says:
+
+> JSON and custom parsers let you do neat things like: 
+> complex pricing, map data to 3D models, auto generate images… 
+
+{::coverage year="2022" month="march" talk="a-parse-parse-pitch-using-j-s-o-n-and-custom-parsers-to-create-efficient-flexible-data-structures" /}
+
+## Afterwards
+
+After the formal part of the meeting, if you want to keep on LRUGging a
+handful of us usually stay on the zoom call for a bit of a chat. It's not
+quite the same as hanging out together IRL, but it's better than nothing
+while things remain pandemic-y. If you have some ideas about an
+alternative approach, then let us know at
+[organisers@lrug.org](mailto:organisers@lrug.org).
+
+## Registration {#apr22registration}
+
+Prior to attending you should familiarise yourself with our
+[README](http://readme.lrug.org/) paying close attention to [the code of
+conduct](http://readme.lrug.org/#code-of-conduct) which applies to all
+attendees, even though we are all in our own little bubbles.
+
+### Secure your place
+
+Even in a virtual world there are limited places for attending the
+meeting so **[you need to register via eventbrite][apr2022-eventbrite]**.
+The link to the zoom meeting will only be sent to registered attendees on
+the day of the meeting, around about 6pm. Keep an eye out for the email
+and check your spam just in case.
+
+Do prepare yourself in advance by [downloading the zoom
+client](https://zoom.us/support/download).
+
+[apr2022-eventbrite]: https://www.eventbrite.com/e/lrug-march-2022-tickets-292085996077
```

Comment:
Link points to the March event.

Reply:
I still haven't published the event on Eventbrite I'll update once that is done